src/init/common.h

```diff
@@ -8,11 +8,11 @@
 #ifndef BITCOIN_INIT_COMMON_H
 #define BITCOIN_INIT_COMMON_H
 
+#include <kernel/include/init/common.h>
```

Comment:
nit: Pretty sure this is illegal according to iwyu?

Reply:
Ah right. Yeah the thought here (in general for the `libbitcoinkernel` project) is for the existing headers to export things from the "kernel-extracted version of headers" so that we don't have too much code churn adding `#include <kernel/*.h>` all over the place. (See my other comment here for more details: https://github.com/bitcoin/bitcoin/pull/25065#discussion_r865910970)

I think a good solution would be to do:
```
#include <kernel/include/init/common.h> // IWYU pragma: export
```

Reply:
Fixed as of: 31c21f3644f6f42f6e4c3c231e95bf0b87ab86f5